addons/queryparams/README.md

```diff
@@ -10,27 +10,55 @@ First, install the addon.
 $ yarn add @storybook/addon-queryparams --dev
 ```
 
-Add this line to your `main.js` file (create this file inside your storybook config directory if needed).
+Register it by adding it in the addons attribute in your `main.js` file (create this file inside your storybook config directory if needed).
 
 ```js
 module.exports = {
   addons: ['@storybook/addon-queryparams'],
 };
 ```
 
+In your story, add the `withQuery` decorator and define the query parameters you want to mock:
 ```js
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { Button } from '@storybook/react/demo';
+import { withQuery } from '@storybook/addon-queryparams';
 
-storiesOf('button', module)
-  .addParameters({
+export default {
+  title: 'Button',
+  component: Button,
+  decorators: [ withQuery ],
```

Comment:
@yannbf can you add prettier to your editor? it will automatically remove the spaces around `withQuery`

Reply:
Sorry I have format on save disabled, given I work on different projects and not all of them use prettier. All done!